packages/editor/src/index.ts

```diff
@@ -16,14 +16,18 @@ import type { PartialDeep, InquirerReadline } from '@inquirer/type';
 type EditorConfig = {
   message: string;
   default?: string;
-  postfix?: string;
```

Comment:
This is going to create a breaking change that risk leaving a fair amount of the community behind. Let's maintain backward compatibility for this option.

Reply:
good point, i will make the change